### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by yichuandoris** [yichuandoris/MI-449-SS17-740-css-spacing-and-layout-T7bJIb#1]
@egillespie Can you take a look at this? It's [hosted here](https://wangyi70-kapm87bvfi5cut7bjib.surge.sh/) and meets the following criteria:

- [x] Demonstrate an understanding of the CSS we've learned so far in the class
- [x] Use Flexbox to create the layout of your page
- [x] Use `margin` and `padding` to create space in and between elements
- [x] Use display `block` and `inline-block` properly
- [x] Have fun and do your best to make it pretty, the result should be something you would show to a future employer

Hi Erik,
I would like to add more elements on it, but just want to make sure my work right now looks right.
My current problem is that I want to move it to the "about" and "contact" information to the right side to the photo. But I have no idea how to make it. 
Of course, we can work on it during the meeting on Wednesday. Thanks!! - Doris

**Comment by egillespie** [comment]
Yep, we can certainly work on this in the afternoon! 😄 I'll send you some ideas in case you want to try some out before we meet up though.

### Avoid style classes that are named after the semantic elements

There are a few cases where you created style classes such as `.header` and `.footer` and applied them to semantic HTML elements of the same name (`header`, `footer`). Instead of adding style classes, you can more simply style the semantic elements directly.

So instead of this:

```html
<header class="header">
  ...
</header

.header {
  // ...
}
```

You can do this:

```html
<header>
  ...
</header>

header {
  // ...
}
```

All I did was remove the `class="header"` and replace `.header` with `header` in the Sass/CSS.

### Idea to move contact/about to right of your photo

If you make the `header` element `display: flex` and move the `h1` element inside your `article` then I think you will see the contact info appear to the right of your photo.

The photo may need to set a `max-height` instead of `width` or `max-width` or else your photo may appear disproportionately tall.

You will also want to set `flex-wrap: wrap` so on small screens your contact info will appear below your photo.

---

Give those a shot, we can talk about it more this afternoon. See ya! 🦄

**Comment by yichuandoris** [comment]
Hi Erik, I've fixed some problems you mentioned, can you have another look at it, thanks. - Doris

**Comment by egillespie** [comment]
This is shaping up nicely! 💃 

When I shrink the browser window so it is not very wide, your photo and info look a little weird. I think setting `margin: 20px auto` on your `img` will take care of this. While you're updating the style for your `img`, you can set `display: inline-block` to satisfy requirement 4. 😉 

I also noticed that "experience" is spelled wrong. >_<

**Comment by yichuandoris** [comment]
Thank you Erik, I've updated those issues including the wrong spelling. - Doris

**Comment by egillespie** [comment]
I was peeking over your shoulder. Looks great! 😄 :shipit:
